github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -489,7 +489,7 @@ periodics:
     testgrid-dashboards: kubevirt-periodics
     testgrid-days-of-results: "60"
   cluster: prow-workloads
-  cron: 0 22, 11 * * *
+  cron: 0 23 * * 5,6
```

Comment:
@oshoval Could you please explain the meaning of these numbers?

Reply:
I know they are the cron format, but what it the meaning to humans?

Reply:
https://crontab.guru
adding comments are prone to get outdated so it doesn't make sense
"“At 23:00 on Friday and Saturday.”

Reply:
> adding comments are prone to get outdated so it doesn't make sense

It was not my intention to request addition of a comment.
But I think adding a human readable explanation to the commit message could be beneficial.

Reply:
We are using cron all over, I think it is an overkill

Reply:
I know the cron format (not by heart though)
I just think that when something is not straight-forward, its better to write what it does.
It will also help the reviewer to verify if what you did is equal to what you meant to do.

Reply:
To ease the reviewer I supplied the string as mentioned above. and not just pasted the site.
I could just add it but imo it isn't a good practice,
it isn't something we are doing and I don't think we should start.